otx/algorithms/common/adapters/mmcv/runner.py

```diff
@@ -68,7 +68,7 @@ def stop(self) -> bool:
         return broadcast_obj[0]
 
     @check_input_parameters_type()
-    def train(self, data_loader: DataLoader, **kwargs):
+    def train(self, data_loader, **kwargs):
```

Comment:
why do you remove type hinting. Can other type dataloader be passed to this function?